pkg/io/storage/storage_test.go

```diff
@@ -0,0 +1,57 @@
+package storage
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestFileVault_SaveData(t *testing.T) {
+
+	fileVault, err := NewFileVault("file")
```

Comment:
Tests are also should be self-cleaning, so there should be code to delete created files.